src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -31,9 +31,9 @@ public void isSamePerson() {
         // null -> returns false
         assertFalse(ALICE.isSamePerson(null));
 
```

Comment:
Given that email no longer has a special significance in equality check, I think some of the test cases below are redundant now. Perhaps enough to test these two only?

* Same name, everything else is different -> true
* Different name, everything else is same -> false

What do you think?

Reply:
i agree, the following 3 would be redundant
* same name, same phone, different attributes -> returns true
* same name, same email, different attributes -> returns true
* same name, same phone, same email, different attributes -> returns true

I will remove these tests and update the commit accordingly